userspace/common/strlcpy.h

```diff
@@ -26,6 +26,7 @@ limitations under the License.
 */
 
 #ifndef HAVE_STRLCPY
+#ifndef __APPLE__
```

Comment:
Why doesn't the cmake check handle this?

Reply:
addressed in 512270666f3ba2db33c669293dcc9e3bea9a7ca8